lib/dialects/abstract/query-generator.js

```diff
@@ -2070,7 +2070,16 @@ class QueryGenerator {
       if (value && value instanceof Utils.SequelizeMethod) {
         value = this.getWhereConditions(value, tableName, factory, options, prepend);
 
-        return value === 'NULL' ? `${key} IS NULL` : [key, value].join(` ${smth.comparator} `);
+        if (value === 'NULL') {
```

Comment:
Coverage is down for these lines, Can you add additional unit test for this case?

Reply:
I just added some to test query-generator.test.js and some for utils.test.js.